urllib3/_backends/_loader.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+from ..backends import Backend as UserSpecifiedBackend
```

Comment:
I'm trying to call out here that this is a user interface and should be handled appropriately. It's also not a "Backend" - it's a recipe for building one, but our users shouldn't need to think about that.